src/main/java/edu/berkeley/ground/db/CassandraClient.java

```diff
@@ -192,6 +276,63 @@ public void delete(List<DbDataContainer> predicates, String table) {
     this.session.execute(statement);
   }
 
+	/**
+   * Deletes a column from a table (sets column to null)
```

Comment:
The documentation was a mistake, the method does actually delete columns. The docs (and method) has been updated